eessi-2023.06-known-issues.yml

```diff
@@ -0,0 +1,4 @@
+- aarch64/neoverse_v1:
+  - netCDF-4.9.2-GCC-12.3.0/GCC-13.2.0:
+      - issue: https://github.com/EESSI/software-layer/issues/425
+      - info: "netCDF intermittent test failures"
```

Comment:
@TopRichard We need separate entries here, because we eventually want to automatically process this YAML file to auto-update a page in the docs that mentions known issues:

```yaml
  - netCDF-4.9.2-gompi-2023a.eb:
      - issue: https://github.com/EESSI/software-layer/issues/425
      - info: "netCDF intermittent test failures"
  - netCDF-4.9.2-gompi-2023b.eb:
      - issue: https://github.com/EESSI/software-layer/issues/425
      - info: "netCDF intermittent test failures"